core/src/test/java/org/springframework/security/access/AccessDeniedExceptionTests.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2002-2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.security.access;
+
+import java.util.Locale;
+
+import org.junit.jupiter.api.Test;
+
+import org.springframework.context.i18n.LocaleContextHolder;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * @author Dayan Kodippily - 17/7/21
+ */
+public class AccessDeniedExceptionTests {
+
+	@Test
+	public void returnsExceptionMessageUsEnglishDefaultLocale() {
+		Locale beforeSystem = Locale.getDefault();
+		Locale.setDefault(Locale.US);
+		Locale beforeHolder = LocaleContextHolder.getLocale();
+		LocaleContextHolder.setLocale(Locale.US);
```

Comment:
Can these steps and the steps at the end of the method be incorporated into an `@BeforeEach` and `@AfterEach`?

Reply:
These changes are added